core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -1083,12 +1089,32 @@ funds.tx.dustAttackTx.popup=This transaction is sending a very small XMR amount
 ####################################################################
 
 support.tab.mediation.support=Mediation
-support.tab.refund.support=Refund
 support.tab.arbitration.support=Arbitration
 support.tab.legacyArbitration.support=Legacy Arbitration
 support.tab.ArbitratorsSupportTickets={0}'s tickets
+support.tab.SignedOffers=signed offers
 support.filter=Search disputes
 support.filter.prompt=Enter trade ID, date, onion address or account data
+support.prompt.signedOffer.penalty.msg=This offer is identified by ID {0}. Are you sure you want to send?\n\n\
+  Other Info:\n\
+  Maker Trade Fee: {1}\n\
+  Reserve Tx Miner Fee: {2}\n\
+  Reserve Tx Hash: {3}\n\
+  Reserve Tx Key Images: {3}\n\
+
+support.contextmenu.penalize.msg=Penalize {0} by publishing reserve tx
+support.prompt.signedOffer.error.msg=Signed Offer record does not exist; contact administrator.
+support.info.submitTxHex=Reserve transaction has been published with the following result:\n
+support.result.hasInvalidInput=Transaction has invalid inputs\n
+support.result.hasInvalidOutput=Transaction has invalid outputs\n
+support.result.hasTooFewOutputs=Transaction has too few outputs\n
+support.result.isDoubleSpend=Transaction resulted in double spend\n
+support.result.isGood=Transaction is good\n
+support.result.isFeeTooLow=Transaction fee too low\n
+support.result.isOverspend=Transaction is over-spend\n
+support.result.isRelayed=Transaction is relayed\n
+support.result.isMixinTooLow=Transaction mixin is too low\n
+support.result.success=Transaction hex has been successfully submitted.
```

Comment:
These messages need removed since the result should not be broken down.

Reply:
Entries removed as shown below:
![Screenshot from 2023-02-28 16-26-18](https://user-images.githubusercontent.com/20237127/221899220-3f39b319-c853-4581-a6e0-5b4378be94e5.png)